js/tji-datasets-explore.js

```diff
@@ -454,7 +473,8 @@ TJIChartView.prototype.create_charts = function() {
     that.state.charts.push(
       new chart_constructor({
         $container: $container, 
-        groupBy: config.group_by, 
+        group_by: config.group_by, 
+        sort_by: config.sort_by,
```

Comment:
Unrelated to your PR: instead of `config` and `chart_configs`, maybe we should call it `props` and `chart_props`, to be consistent?

Reply:
🤷‍♀️ you can rewrite if you want. We call it chart_configs in the ChartView constructor.